sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/service/UserService.java

```diff
@@ -18,6 +18,7 @@
 	private Map<String, User> email2User = new HashMap<String, User>();
 
 	public User create(String email, String password) {
+		String tmp;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "tmp" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)